pypowsybl/network.py

```diff
@@ -4374,3 +4374,9 @@ def get_extensions_names() -> _List[str]:
         the names of the available extensions
     """
     return _pp.get_extensions_names()
+
+def create_branch_feeder_bays(n: Network, bus_bar_section_id_1: str, bus_bar_section_id_2: str, position_order_1: int, position_order_2: int, direction_1: str, direction_2, df_new_line: _DataFrame = None, **kwargs: _ArrayLike) -> None:
```

Comment:
Like for injections, I would rather use only dataframes as arguments, to be able to create multiple elements at once.

Also, I think position should be optional since it does not exist in all networks (should be natural if we use dataframes).

Reply:
All arguments has been moved into the dataframe. There is only the substation for twt creation that remain separated.

Reply:
Actually, I think the substation argument is not useful (it's redundant): 
we should be able to retrieve it from the bus or busbars that are defined. Is it possible to implement this ?